src/services/auth.service.ts

```diff
@@ -51,11 +58,13 @@ export async function login(
     const accessToken = generateAccessToken(user as IUser);
     const refreshToken = generateRefreshToken(user as IUser);
 
-    return { accessToken, refreshToken };
+    console.log(user._id)
```

Comment:
Clarify the purpose of console logging the user ID during the login process; consider if this is necessary or should be removed.

Reply:
deleted)

Reply:
OK